### ISSUE TITLE
tftocaffe flatten

### ISSUE THREAD

**Issue by xubin19939** [Microsoft/MMdnn#596]
Platform (like ubuntu 16.04/win10):ubuntu 16.04

Python version:2.7.12

Source framework with version (like Tensorflow 1.4.1 with GPU):tf1.8

Destination framework with version (like CNTK 2.3 with GPU):GPU

Pre-trained model path (webpath or webdisk path):https://github.com/xubin19939/tf-resnettocaffe

Running scripts:
mmconvert -sf tensorflow -iw scene.pb --inNodeName inputs --inputShape 128,128,3 --dstNodeName Softmax -df caffe -om TF_scene

errors:
I0226 11:49:44.609355 26674 net.cpp:84] Creating Layer pool_9_AvgPool_crop
I0226 11:49:44.609359 26674 net.cpp:406] pool_9_AvgPool_crop <- pool_9_AvgPool
I0226 11:49:44.609364 26674 net.cpp:406] pool_9_AvgPool_crop <- DummyData3
I0226 11:49:44.609367 26674 net.cpp:380] pool_9_AvgPool_crop -> pool_9_AvgPool_crop
I0226 11:49:44.609376 26674 net.cpp:122] Setting up pool_9_AvgPool_crop
I0226 11:49:44.609380 26674 net.cpp:129] Top shape: 1 64 5 5 (1600)
I0226 11:49:44.609382 26674 net.cpp:137] Memory required for data: 8984448
I0226 11:49:44.609385 26674 layer_factory.hpp:77] Creating layer flatten_flatten_Reshape
I0226 11:49:44.609390 26674 net.cpp:84] Creating Layer flatten_flatten_Reshape
I0226 11:49:44.609393 26674 net.cpp:406] flatten_flatten_Reshape <- pool_9_AvgPool_crop
I0226 11:49:44.609400 26674 net.cpp:380] flatten_flatten_Reshape -> flatten_flatten_Reshape
F0226 11:49:44.609416 26674 reshape_layer.cpp:22] Check failed: inferred_axis_ == -1 (0 vs. -1) new shape contains multiple -1 dims; at most a single (1) value of -1 may be specified
*** Check failure stack trace: ***
Aborted (core dumped)

 It seems there are something wrong with flatten?please help me ,thanks
